exp-models/addon/models/experiment.js

```diff
@@ -139,6 +139,21 @@ const Experiment = DS.Model.extend(JamModel, {
         maxDays = maxDays || Number.MAX_SAFE_INTEGER;
         return minDays <= age && age <= maxDays;
     },
+    isOldEnough(participant) {
```

Comment:
I notice that a lot of this code is similar to `isEligible`, which could add challenges for maintenance later. Would you object if I sent a PR to your branch, demonstrating an idea to help simplify this?

(cool github tidbit: you can send a PR to another PR! The message will show up in your fork, and if you merge it, my commits would automatically become part of this here thread. It's very meta)